<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynoChartTaskPanel</class>
 <widget class="QWidget" name="DynoChartTaskPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plot Dyno Chart</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>231</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QGroupBox" name="grpbx_master">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>229</width>
     <height>289</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="grpbx_sprdsheet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Plots to View</string>
    </property>
    <widget class="QCheckBox" name="chkbx_FntMotorDyno">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Front Motor Dyno Chart</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbx_RrMotorDyno">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rear Motor Dyno Chart</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbx_AxleTorque">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Axle Torque</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbx_VehForces">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Vehicle Forces at Play</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbx_WheelPwr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wheel Power - Battery Limit</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbx_AccelTraction">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Acceleration - Traction Est.</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbx_FnBatteryCapa">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Front Battery Capability</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkbx_RrBatteryCapa">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Rear Battery Capability</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Manage Data Rows</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Adjust Rows to Plot</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinbx_Rows">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Default: 16. Range: 2-50</string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>52</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
